internal/tableland/tableland.go

```diff
@@ -58,6 +58,16 @@ type ValidateCreateTableResponse struct {
 	StructureHash string `json:"structure_hash"`
 }
 
+// ValidateWriteQueryRequest is a ValidateWriteQuery request.
+type ValidateWriteQueryRequest struct {
+	Statement string `json:"statement"`
+}
+
+// ValidateWriteQueryResponse is a ValidateWriteQuery response.
+type ValidateWriteQueryResponse struct {
+	TableID string `json:"table_id"`
```

Comment:
Since TableID is a big.Int it seemed natural to convert to string, but I'm not sure if there's a better approach

Reply:
Yes, this is correct. `string` is fine since the number can be theoretically big, and in JS numbers are floats.